src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { render } from "react-dom";
+
+function Hi() {
```

Comment:
use an arrow function instead.

```suggestion
const Hi = () => <p>Hi.</p>
```